packages/android_intent_plus/pubspec.yaml

```diff
@@ -14,15 +14,18 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
-  platform: '>=2.0.0 <=3.0.0'
-  meta: ^1.0.5
+
+  platform: ^3.0.0-nullsafety.4
+  meta: ^1.3.0-nullsafety.6
```

Comment:
the migration guidelines recommend leaving out `.x` suffixes to make version solving more flexible